controllers/articles.controllers.js

```diff
@@ -0,0 +1,10 @@
+const { selectArticleById } = require("../models/articles.models");
+
+exports.getArticleById = (req, res, next) => {
+  const article_id = req.params.article_id;
+  selectArticleById(article_id)
+    .then((article) => {
+      res.status(200).send(article);
+    })
```

Comment:
can we send back the article on the key of article?

Reply:
I almost did this but then convinced myself that the ticket specified that it returns an object with certain properties. Will update to match what we generally do.